packages/sdk/src/across/feeCalculator.ts

```diff
@@ -11,6 +17,13 @@ export interface RateModel {
   R2: BigNumberish; // R_0+R_1+R_2 is the interest rate charged at 100% utilization
 }
 
+export interface Web3RateModel {
+  UBar: Web3BN; // denote the utilization kink along the rate model where the slope of the interest rate model changes.
+  R0: Web3BN; // is the interest rate charged at 0 utilization
+  R1: Web3BN; // R_0+R_1 is the interest rate charged at UBar
+  R2: Web3BN; // R_0+R_1+R_2 is the interest rate charged at 100% utilization
+}
```

Comment:
it might clean things up to treat rate models as strings in wei rather than big numbers, that way you can import them from the constants file and pass them into the calculator as is. I adjusted the calculator to use BigNumberish which allows for strings, numbers or bignumbers.  Most of your definitions of the rate models use `toBNWei` which you could probably just change to `toWei` and it should work unless you are doing math on this outside of the sdk

Reply:
ye, that's a great point. dealing with two different types of big number (web3 & ethers/BigNumber.js) is a _huge_ pain. I'll move to using strings ideally

Reply:
please take a look at how I do it now. I think it's much cleaner now